.travis.yml

```diff
@@ -1,3 +1,6 @@
+git:
+    quiet: true
```

Comment:
Do you have any reference for this property? Did a quick google and didn't find anything quickly.